### PR TITLE
Feature/replace array device and raid6 support

### PR DESCRIPTION
**Pull request by manoj22-k**
Added replace array feature in Mtool
Support raid6 option in array creation

### CODE REVIEW

File: src/mtool/api/rest/app.py [hunk 1]
```diff
@@ -927,6 +927,35 @@ def removeSpareDisk(current_user):
     return make_response(res, 500)
 
 
+# Replace Array Device
+@app.route('/api/v1/array/<array_name>/replace', methods=['POST'])
+def replace_arr_device(array_name):
+    try:
+        body_unicode = request.data.decode('utf-8')
+        body = json.loads(body_unicode)
+        print("body", body, array_name)
```
Comment (aswinssir) [inline src/mtool/api/rest/app.py:957]: Please remove these print statements which are used for debugging

File: src/mtool/api/rest/app.py [hunk 2]
```diff
@@ -948,6 +948,35 @@ def removeSpareDisk(current_user):
...
```
Comment (aswinssir) [inline src/mtool/api/rest/app.py:957]: Please remove the print statements that are written for debugging purposes

File: src/mtool/api/rest/app.py [hunk 3]
```diff
@@ -1253,13 +1288,17 @@ def get_arrays(current_user):
...
                 a_info['volumecount'] = len(vol_list)
                 a_info["arrayname"] = res["result"]["data"]["name"]
                 a_info["status"] = array["status"]
                 a_info["situation"] = res["result"]["data"]["situation"]
                 a_info["state"] = res["result"]["data"]["state"]
-                a_info["writeThroughEnabled"] = res["result"]["data"]["writeThroughEnabled"]
+                if "writeThroughEnabled" in res["result"]["data"]:
+                    a_info["writeThroughEnabled"] = res["result"]["data"]["writeThroughEnabled"]
+                else:
+                    a_info["writeThroughEnabled"] = True
```
Comment (aswinssir) [inline src/mtool/api/rest/app.py:1301]: I think writeThroughenabled should be false by default

File: src/mtool/api/rest/app.py [hunk 4]
```diff
@@ -948,6 +948,35 @@ def removeSpareDisk(current_user):
...
+        print("body", body, array_name)
+        device = body['device']
+        return_msg = {}
+        res = dagent.replace_array_device(array_name, device)
+        if res.status_code == 200:
+            res = res.json()
+            if res["result"]["status"]["code"] == 0:
+                return toJson(res)
+        else:
+            print("spare error", res.json())
```
Comment (vishal-shakya-ssir) [inline src/mtool/api/rest/app.py:966]: Please remove this print statement

**Review by aswinssir — APPROVED** [review]
LGTM

**Review by vishal-shakya-ssir — APPROVED** [review]
LGTM !

File: src/mtool/api/rest/app.py (hunk 4 above)
Comment (manoj22-k) [reply src/mtool/api/rest/app.py]: Removed thanks

File: src/mtool/api/rest/app.py (hunk 3 above)
Comment (manoj22-k) [reply src/mtool/api/rest/app.py]: Updated the default value to False.

**Pull request merged**